### PR TITLE
scroll position and lint issues

### PR DESCRIPTION
**Pull request by anil-kk**
Added functionality for getting scroll position by emitting **scrollChanged** event with scrollposition and fixed issues with lint. lint is also complaining about the selector for directive, but I disabled lint for this line, if we change selector it might break applications that are already using slimscroll.

### CODE REVIEW

File: src/app/ngx-slimscroll/directives/slimscroll.directive.ts [hunk 1]
```diff
@@ -21,28 +22,29 @@ import 'rxjs/add/operator/takeUntil';
...
+  outQuart: (t: number) => 1 - (--t) * t * t * t,
+  inOutQuart: (t: number) => t < .5 ? 8 * t * t * t * t : 1 - 8 * (--t) * t * t * t,
+  inQuint: (t: number) => t * t * t * t * t,
+  outQuint: (t: number) => 1 + (--t) * t * t * t * t,
+  inOutQuint: (t: number) => t < .5 ? 16 * t * t * t * t * t : 1 + 16 * (--t) * t * t * t * t
 };
 
 @Directive({
-  selector: '[slimScroll]',
+  selector: '[slimScroll]', // tslint:disable-line
```
Comment (jkuri) [inline src/app/ngx-slimscroll/directives/slimscroll.directive.ts]: you can set `"prefix": ""` in `.angular-cli.json` but this is okay too.

File: src/app/ngx-slimscroll/directives/slimscroll.directive.ts [hunk 2]
```diff
@@ -21,28 +22,29 @@ import 'rxjs/add/operator/takeUntil';
...
   exportAs: 'slimScroll'
 })
 export class SlimScrollDirective implements OnInit, OnDestroy {
   @Input() options: SlimScrollOptions;
   @Input() scrollEvents: EventEmitter<SlimScrollEvent>;
+  @Output('scrollChanged') scrollChanged = new EventEmitter<any>();
```
Comment (jkuri) [inline src/app/ngx-slimscroll/directives/slimscroll.directive.ts]: you can be more specific about the event properties here
```ts
@Output('scrollChanged') scrollChanged = new EventEmitter<{ scrollPosition: number }>();
```

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,9 +1,10 @@
 <div class="hero is-fullheight">
   <div class="hero-body">
     <div class="container">
+
       <div class="columns is-multiline">
         <div class="column is-6 is-offset-3">
```
Comment (jkuri) [inline src/app/app.component.html]:
```html
<div class="column is-6 is-offset-3 has-text-centered">
```
here to have it center aligned.

File: src/app/app.component.html [hunk 4]
```diff
@@ -49,6 +50,11 @@ <h1>What is Lorem Ipsum?</h1>
           </div>
         </div>
       </div>
+      <div class=columns>
+          <div class="column is-6 is-offset-6">
```
Comment (jkuri) [inline src/app/app.component.html]:
```html
<div class="column is-6 is-offset-3 has-text-centered">
```
here to have it center aligned.

**Comment by anil-kk** [comment]
@jkuri added little extra functionality, state of scroll can be very useful when building applications, one example use case: infinite scroll behavior can be achieved, by loading more items when scroll is ending.

**Comment by jkuri** [comment]
thank you @anil-kk. very nice PR!

**Pull request merged**